src/object/srv_obj.c

```diff
@@ -621,10 +741,15 @@ obj_local_rw(crt_rpc_t *rpc, struct ds_cont_hdl *cont_hdl,
 	uint32_t		tag = dss_get_module_info()->dmi_tgt_id;
 	daos_handle_t		ioh = DAOS_HDL_INVAL;
 	struct bio_desc		*biod;
+	struct daos_oclass_attr *oca =
+				 daos_oclass_attr_find(orw->orw_oid.id_pub);
```

Comment:
style: if this line is two long, we might just call the function in the function body, instead of putting it together with declare